### DISCUSSION TITLE
How are sessions removed?

### DISCUSSION THREAD

**Discussion by esjch2020** [microsoft/azure-boards-estimate#195] [Q&A]
We recently had some disappear but have no idea how they can even be removed. Is there a way to determine the history of removed sessions as well? If not, this would be helpful. 

There are several sessions that no longer have the Query, so we're stuck with this session with an inability to remove it. It would be nice to have a remove button to clear up old Sessions.

**Comment by hangy** [comment]
I think you can click "end session" from the context (?) menu of the session list/overview. (Menu with the three dots on each card.)

IIRC, there's no way to see ended sessions, because they're deleted completely.
